Source/Microsoft.Teams.Apps.FAQPlusPlus/Properties/Strings.resx

```diff
@@ -601,4 +613,76 @@ I am your friendly Q&amp;A bot that helps you provide support to people who inte
     <value>How helpful is this app? (Required)</value>
     <comment>Feedback for App</comment>
   </data>
+  <data name="BotCommandAskExpert" xml:space="preserve">
```

Comment:
These need to match the translations in the manifest, right? Please add a comment to this resource noting that constraint, since that is not obvious.

Reply:
Same with the other strings with this constraint.

Reply:
Added comment for bot commands

Reply:
"Please add a comment to this resource noting that constraint, since that is not obvious."
I see we added comments, but it doesn't mention the important part, which is that **these strings need to match the values in the app manifests**.

Reply:
Hello @aosolis,
Earlier I have given comment as : bots[0].commandLists[0].commands[1].title in manifest's translated files should match with this value..
Updated it as: Bot command Ask an expert. This needs to be match with manifest's command "Ask an expert" in translated json file.

Let me know if this still doesn't sound clear.